scanner/cmd/updater/update.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"context"
+	"flag"
+
+	"github.com/quay/zlog"
+	"github.com/stackrox/stackrox/scanner/v4/updater"
+)
+
+func main() {
+	outputDir := flag.String("outputDir", "", "Specify the output directory")
+	flag.Parse()
```

Comment:
If we don't specify `-outputDir` when calling this manually (let's say, if I am testing), the tool will fail with `no such file or directory`.  I think what you want is:

```suggestion
	if len(os.Args[1:]) == 0 {
		log.Fatal("Missing argument to the output directory.")
	}
	outputDir := os.Args[1]
```